tribits/examples/TribitsExampleProject2/packages/package1/CMakeLists.txt

```diff
@@ -1,4 +1,57 @@
-tribits_package(Package1)
-add_subdirectory(src)
-tribits_add_test_directories(test)
-tribits_package_postprocess()
+cmake_minimum_required(VERSION 3.23.0 FATAL_ERROR)
+
+macro(include_raw_cmake_build)
+  if (NOT COMMAND tribits_project)
+    include("${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.raw.cmake"
+       NO_POLICY_SCOPE)
+    return()
+  endif()
+endmacro()
+
+if (COMMAND tribits_package)
```

Comment:
That is a good point.  In my official documentation for this use case [here](https://tribitspub.github.io/TriBITS/users_guide/index.html#how-to-put-a-tribits-and-raw-cmake-build-system-side-by-side), I used the variable `${PROJECT_NAME}_TRIBITS_DIR`.  But it may be better to define a variable that has this specific meaning `<PackageName>_BEING_PROCESSED_AS_TRIBITS_INTERNAL_PACKAGE` so that there is no ambiguity like:

```
if (Package1_BEING_PROCESSED_AS_TRIBITS_INTERNAL_PACKAGE)
  ...
else()
  ...
endif()
```

What do you think?

Reply:
We can tweak this later.